core/templates/dev/head/components/RatingDisplayDirective.js

```diff
@@ -16,88 +16,91 @@
-            ratingValue < $scope.stars[i].value - 0.25 ? 'fa-star-half-o' :
-            'fa-star');
+        var displayValue = function(ratingValue) {
+          for (var i = 0; i < $scope.stars.length; i++) {
+            $scope.stars[i].cssClass = (
+              ratingValue === undefined ? 'fa-star-o' :
+              ratingValue < $scope.stars[i].value - 0.75 ? 'fa-star-o' :
+              ratingValue < $scope.stars[i].value - 0.25 ? 'fa-star-half-o' :
+              'fa-star');
 
-          if ($scope.status === STATUS_ACTIVE &&
-              ratingValue >= $scope.stars[i].value) {
-            $scope.stars[i].cssClass += ' oppia-rating-star-active';
+            if ($scope.status === STATUS_ACTIVE &&
+                ratingValue >= $scope.stars[i].value) {
+              $scope.stars[i].cssClass += ' oppia-rating-star-active';
+            }
           }
-        }
-      };
+        };
 
-      displayValue($scope.ratingValue);
-      $scope.$watch('ratingValue', function() {
         displayValue($scope.ratingValue);
-      });
+        $scope.$watch('ratingValue', function() {
+          displayValue($scope.ratingValue);
+        });
 
-      $scope.clickStar = function(starValue) {
-        if ($scope.isEditable && $scope.status === STATUS_ACTIVE) {
-          $scope.status = STATUS_RATING_SET;
-          $scope.ratingValue = starValue;
-          displayValue(starValue);
-          $scope.onEdit(starValue);
-        }
-      };
-      $scope.enterStar = function(starValue) {
-        if (
-            $scope.isEditable &&
-            ($scope.status === STATUS_ACTIVE ||
-              $scope.status === STATUS_INACTIVE)) {
-          $scope.status = STATUS_ACTIVE;
-          displayValue(starValue);
-        }
-      };
-      $scope.leaveArea = function() {
-        $scope.status = STATUS_INACTIVE;
-        displayValue($scope.ratingValue);
-      };
+        $scope.clickStar = function(starValue) {
+          if ($scope.isEditable && $scope.status === STATUS_ACTIVE) {
+            $scope.status = STATUS_RATING_SET;
+            $scope.ratingValue = starValue;
+            displayValue(starValue);
+            $scope.onEdit(starValue);
+          }
+        };
+        $scope.enterStar = function(starValue) {
+          if (
+              $scope.isEditable &&
+              ($scope.status === STATUS_ACTIVE ||
+                $scope.status === STATUS_INACTIVE)) {
+            $scope.status = STATUS_ACTIVE;
+            displayValue(starValue);
+          }
+        };
+        $scope.leaveArea = function() {
+          $scope.status = STATUS_INACTIVE;
+          displayValue($scope.ratingValue);
+        };
 
-      $scope.getCursorStyle = function() {
-        return 'cursor: ' + ($scope.isEditable ? 'pointer' : 'auto');
-      };
-    }]
-  };
-}]);
+        $scope.getCursorStyle = function() {
+          return 'cursor: ' + ($scope.isEditable ? 'pointer' : 'auto');
+        };
+      }]
+    };
+  }]);
```

Comment:
maybe drop ]); to the next line so that the brackets line up with the opening ones?

Reply:
done.